app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
+    redirect_to "/person_not_found"
+  end
+
+  #----------------- SHOW A PERSON'S TASKS --------------------
+  def tasks
+    @person = Person.find(params[:id])
+    @tasks = Task.all.where(person_id: params[:id])
+
+    @people = [{
+        id: @person.id,
+        name: @person.name,
+        count: @person.tasks.where(date_complete: nil).count
+      }]
```

Comment:
Okay, I've implemented this, and it made the code on the `.erb` files and the controllers much easier to read.